build/Dockerfile.odh-multi

```diff
@@ -0,0 +1,33 @@
+# Default to opendatahub-operator for the sake of odh autobuild
+ARG binary_name=opendatahub-operator
+FROM registry.access.redhat.com/ubi8/go-toolset:1.13.15 AS builder
+ARG binary_name
+USER root
+WORKDIR /scratch
+COPY . .
```

Comment:
truthfully, this copy could probably be pruned to just the necessary files to build

Reply:
Experimented with this copy ... something strange going on here, without "COPY . ." the build fails during "go mod vendor". Something subtle about the structure or perms, idk .. not worth figurout out.